components/link.js

```diff
@@ -5,7 +5,7 @@ import NextLink from 'next/link'
 const InternalLink = React.forwardRef(function InternalLink({ children, className, ...props }, ref) {
   return (
     <NextLink { ...props }>
-      <a className={ className } ref={ ref }>{ children }</a>
+      <a className={ className } ref={ ref } {...props}>{ children }</a>
```

Comment:
do we want props distributed into NextLink and a? an incoming prop like `key`, for example, we would not want this behavior, so it feels like we want to be a bit careful here.